src/pages/iou/request/step/IOURequestStepMerchant.js

```diff
@@ -71,7 +102,25 @@ function IOURequestStepMerchant({
      * @param {String} value.moneyRequestMerchant
      */
     const updateMerchant = (value) => {
-        IOU.setMoneyRequestMerchant_temporaryForRefactor(transactionID, value.moneyRequestMerchant);
+        const newMerchant = value.moneyRequestMerchant.trim();
```

Comment:
Adding this on this line since the comment below was resolved, but there should be blank line before the comment
```suggestion
        const newMerchant = value.moneyRequestMerchant.trim();
        
```